als-oracle-pathfinder/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for als-oracle-pathfinder.
+# This is a YAML-formatted file.
+
+# Declare global configurations
+global:
+  config:
+    ## Pod scheduling preferences.
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ##
+    affinity: {}
+
+    ## Node labels for pod assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+    nodeSelector: {}
+
+    ## Set toleration for schedular
+    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+    tolerations: []
+
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: mojaloop/als-oracle-pathfinder
+  tag: v8.3.0
+  pullPolicy: Always
+  imagePullSecrets:
+    name: dock-casa-secret
+
+readinessProbe:
+  enabled: true
+  httpGet:
+    path: /
+  initialDelaySeconds: 45
+  periodSeconds: 15
+
+livenessProbe:
+  enabled: true
+  httpGet:
+    path: /
+  initialDelaySeconds: 45
+  periodSeconds: 15
+
+config:
+  db_type: 'mysql'
+  db_driver: 'mysql'
+  db_host: '$release_name-centralledger-mysql'
+  db_port: '3306'
+  db_user: 'central_ledger'
+  db_password: 'oyMxgZChuu'
+  db_database:
+    central_ledger: 'central_ledger'
+    account_lookup: 'account_lookup'
+
+  pathfinder:
+    host: '156.154.59.224'
+    port: 7007
+    queryTimeoutMs: 10000
+    tls:
+      # rejectUnauthorized should be true in production
+      rejectUnauthorized: false
+      # The following paths are all relative to this directory containing the values file
+      client_cert_path: '/opt/als-oracle-pathfinder/secrets/client_cert.pem'
```

Comment:
All these certs should be loaded from a secret(s), and then mounted into the correct location.

Reply:
ok

Reply:
https://github.com/mojaloop/helm/pull/218/files#diff-ba8c6ec03ed905aae6f9eebc36277168